packages/bruno-app/src/components/RunnerResults/index.jsx

```diff
@@ -124,21 +128,25 @@ export default function RunnerResults({ collection }) {
   }
 
   return (
-    <StyledWrapper ref={listWrapperRef} className="px-4 pb-4 flex flex-grow flex-col relative">
-      <div className="flex flex-row mt-6 mb-4">
-        <div className="font-medium title flex items-center">
+    <StyledWrapper className="px-4 pb-4 flex flex-grow flex-col relative">
+      <div className="flex flex-row">
+        <div className="font-medium my-6 title flex items-center">
           Runner
           <IconRun size={20} strokeWidth={1.5} className="ml-2" />
         </div>
         {runnerInfo.status !== 'ended' && runnerInfo.cancelTokenUid && (
-          <button className="btn ml-6 btn-sm btn-danger" onClick={cancelExecution}>
+          <button className="btn ml-6 my-4 btn-sm btn-danger" onClick={cancelExecution}>
             Cancel Execution
           </button>
         )}
       </div>
-      <div className="flex flex-1">
+      <div
+        className="flex flex-col overflow-y-auto"
```

Comment:
Changing the classes from `flex flex-1` to `flex flex-col overflow-y-auto` messed up the response styling, because of `flex-col`:

![image](https://github.com/usebruno/bruno/assets/39559178/9f751684-726f-408f-b0bb-d88f88f0537e)